domains/artix.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "alex1028199",
+        "email": "yu1234u73f.com@gmail.com"
+    },
+    "record": {
+        "URL": "google.com"
```

Comment:
Serves no purpose as it redirects to google.c

Reply:
I might change it later i just want see domain

Reply:
Point it to 51.68.194.254:3116